src/vcpkg/binarycaching.cpp

```diff
@@ -2166,11 +2166,20 @@ namespace vcpkg
         m_config.read.push_back(std::move(provider));
     }
 
-    std::vector<CacheAvailability> ReadOnlyBinaryCache::precheck(View<InstallPlanAction> actions)
+    void ReadOnlyBinaryCache::clear_cache()
```

Comment:
It seems very wrong that this is necessary; different sets of enabled features should result in different ABI SHAs. If they are not that is a serious bug that must be addressed.

Reply:
Oh, it isn't "clearing" the cache, it's that we blew away 'packages' so it isn't actually restored anymore.